index.bs

```diff
@@ -446,6 +451,38 @@ A {{DatagramDuplexStream}} object has the following internal slots.
 :: The getter steps are:
      1. Return [=this=].`[[Writable]]`.
 
+: <dfn for="DatagramDuplexStream" attribute>incomingMaxAge</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=].
+:: The setter steps are:
+     1. Let |value| be the given value.
+     1. If |value| > 0:
+       1. Set [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=] to |value|.
+
+: <dfn for="DatagramDuplexStream" attribute>outgoingMaxAge</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[OutgoingDatagramsExpirationDuration]]=].
+:: The setter steps are:
+     1. Let |value| be the given value.
+     1. If |value| > 0:
+       1. Set [=this=]'s [=[[OutgoingDatagramsExpirationDuration]]=] to |value|.
+
+: <dfn for="DatagramDuplexStream" attribute>incomingHighWaterMark</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[IncomingDatagramsHighWaterMark]]=].
+:: The setter steps are:
+     1. Let |value| be the given value.
+     1. If |value| >= 0:
```

Comment:
Maybe use `≥`? Or maybe not since I don't think we're using it elsewhere.

Reply:
Thanks, done. I found it's used in WebIDL.